wix-media-v2/wix-media-v2/Files/guides/upload-api.md

```diff
@@ -82,6 +82,8 @@ async function uploadMyFile(uploadUrl, fileContent) {
 }
 ```
 
+## Add Uploaded Media
```

Comment:
Move this section to the top of the document, so it's sitting in essentially what we'd call the endpoint description. That way it's consistent with how we document every other endpoint and function.

Reply:
This is how I updated it, but I don't think its necessary since it does say it was added to the Media Manager at the top of the document.

```### Use uploaded media in your site
After successfully making the REST HTTP request and receiving a valid [response](#Response), the media has been successfully uploaded to your [Media Manager](https://support.wix.com/en/article/wix-media-about-the-media-manager). You can access the content from there and insert it into your site using the UI. 

The data in the response object, however is not directly compatible with your Wix site. To add the uploaded media files to the frontend of your site using code, call [`getFileDescriptor()`](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor). Pass `file.id` from the Upload API's [response object](#Response) as its only parameter. Retrieve the valid Wix URI from the `Media` property in the returned `FileDescriptor` object. Set it to the `src` property of your site's elements.  ```